features/device-config/ssh-scripting/src/main/resources/META-INF/opennms/component-dao.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:context="http://www.springframework.org/schema/context"
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xmlns:onmsgi="http://xmlns.opennms.org/xsd/spring/onms-osgi"
+       xsi:schemaLocation="
+		http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.2.xsd
+		http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx-4.2.xsd
+		http://xmlns.opennms.org/xsd/spring/onms-osgi http://xmlns.opennms.org/xsd/spring/onms-osgi.xsd
+	">
+
+    <context:annotation-config />
+    <tx:annotation-driven />
+    <bean id="sshScriptingService" class="org.opennms.features.deviceconfig.sshscripting.impl.SshScriptingServiceImpl"/>
+</beans>
```

Comment:
Just out of curiosity: Is this the recommended way to have the `SshScriptingService` defined twice, once in `component-dao.xml` and once in `blueprint.xml`?

Reply:
blueprint is needed for minion and `component-dao.xml` on OpenNMS